extensions/exchanges/bitfinex/exchange.js

```diff
@@ -48,11 +48,12 @@ module.exports = function container (get, set, clear) {
     },
 
     getTrades: function (opts, cb) {
-      var func_args = [].slice.call(arguments)
+//      var func_args = [].slice.call(arguments)
```

Comment:
this is getting quite ugly, if it's not necessary then please just remove it, we have git history